back/engines/commercial/bulk_import_ideas/spec/services/parsers/idea_plain_text_parser_service_spec.rb

```diff
@@ -656,6 +680,79 @@
 
       expect(docs).to eq result
     end
+
+    it 'detects field names over multiple lines' do
```

Comment:
It's really great you've added so many tests! I guess these tests have more or less all edge cases from the real PDF files that we've seen?

I'm also wondering what if, at some point, we, let's say, train Document AI a bit, use ChatGPT for text recognition, or do something else. In this case, we'll still want to prevent all regressions. But we could not use these specs for it. In the ideal world, we could test some high-level service by uploading real PDF files to any service we use and then check that all fields are recognised correctly. We cannot do it of course. But we could at least upload all these PDF files (e.g., here https://www.notion.so/citizenlab/Manual-testing-d50d02b57cf742a2815d00033928c622), and add links to them to these specs. So, when we change this processing again, it'll be easier to test all edge cases and avoid regression.
```suggestion
    # Google Document AI response for this file https://notion.so/link-to-this-pdf-file
    it 'detects field names over multiple lines' do
```